languages/english.js

```diff
@@ -121,7 +121,7 @@ const warnings = {
 };
 
 const sast_warnings = {
-  ast_error: "An error occured when parsing the JavaScript code with meriyah. It mean that the conversion from string to AST as failed. If you encounter such an error, please open an issue here.",
+  parsing_error: "An error occured when parsing the JavaScript code with meriyah. It mean that the conversion from string to AST as failed. If you encounter such an error, please open an issue here.",
```

Comment:
```suggestion
  parsing_error: "An error occured when parsing the JavaScript code with meriyah. It mean that the conversion from string to AST has failed. If you encounter such an error, please open an issue here.",
```

I guess this was a typo, maybe we could benefit to fix it within this PR ?